src/navigation/RegisterENSNavigator.js

```diff
@@ -34,9 +34,9 @@ export default function RegisterENSNavigator() {
 
   return (
     <SlackSheet
-      additionalTopPadding
       contentHeight={contentHeight}
       height="100%"
+      removeTopPadding
```

Comment:
maybe this was the cause of ![image](https://user-images.githubusercontent.com/12115171/153499458-d80d2934-d3eb-4b78-95dc-4ad625a8ffba.png)?